src/test/java/com/puppycrawl/tools/checkstyle/MainTest.java

```diff
@@ -70,6 +70,7 @@
         + " -j,--javadocTree            Print Parse tree of the Javadoc comment%n"
         + " -J,--treeWithJavadoc        Print full Abstract Syntax Tree of the file%n"
         + " -o <arg>                    Sets the output file. Defaults to stdout%n"
+        + " -omitIgnoredModules <arg>   Sets ability to not run modules that are ignored.%n"
```

Comment:
Do we need short form for this ?
Short forms exists only to simplify typing for very frequent arguments. I doubt that this option will be used by users, it useful only for our testing tool.